stdlib/public/runtime/SwiftObject.mm

```diff
@@ -1365,7 +1365,13 @@ static bool usesNativeSwiftReferenceCounting_nonNull(
 void swift_objc_swift3ImplicitObjCEntrypoint(id self, SEL selector,
                                              const char *filename,
                                              size_t filenameLength,
-                                             size_t line, size_t column) {
+                                             size_t line, size_t column,
+                                             std::atomic<bool> *didLog) {
```

Comment:
I thought so, although I can't find any proof of that. Regardless, it's likely to hold everywhere that it matters for us. It's fine.